docs/rules/spaced-comment.md

```diff
@@ -175,6 +175,12 @@ Examples of **incorrect** code for this rule with the `"always"` option combined
 /*-+-+-+-+-+-+-+*/
 ```
 
+```js
+/* eslint spaced-comment: ["error", "always", { "line": { "exceptions": ["*"] } }] */
+
+/******** COMMENT *******/
```

Comment:
Can we add the corresponding correct code snippet to the correct examples below? For example:

```js

/***************/


/******** 
COMMENT 
*******/
```